explorer/interpreter/value.h

```diff
@@ -784,7 +807,9 @@ class NominalClassType : public Value {
  private:
   Nonnull<const ClassDeclaration*> declaration_;
   Nonnull<const Bindings*> bindings_ = Bindings::None();
-  std::optional<Nonnull<const NominalClassType*>> base_;
+  const std::optional<Nonnull<const NominalClassType*>> base_;
+  const VTable vtable_;
+  int hierarchy_level_ = 0;
```

Comment:
```suggestion
  int hierarchy_level_;
```

This should always be set, shouldn't it?

Reply:
Done